Viper.xctemplate/ViewController/___FILEBASENAME___ViewController.swift

```diff
@@ -1,23 +1,35 @@
 import UIKit
 
 protocol ___VARIABLE_MODULENAME___ViewProtocol: ___VARIABLE_MODULENAME___Protocol {
-    func setPresenter(_ presenter: ___VARIABLE_MODULENAME___PresenterProtocol)
+    
 }
 
 final class ___VARIABLE_MODULENAME___ViewController: UIViewController {
-    private var presenter: ___VARIABLE_MODULENAME___PresenterProtocol?
+    private let presenter: ___VARIABLE_MODULENAME___PresenterProtocol
+    
+    /*
+        Hint: This initializer won't work if you use a storyboard
+        for this view controller
+        Solution: Use the template again and enable the `Use storyboard` checkbox
+    */
+    init(presenter: ___VARIABLE_MODULENAME___PresenterProtocol) {
+        self.presenter = presenter
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("Use init(presenter:) instead")
```

Comment:
Maybe we could add the information giving in the comment above to the actual user be applying changes to fatalError message? What do you think about that. 
"Use init(presenter:)" could be misleading somehow.

Reply:
True 👍  Done.